test/integration/backward_compatible/parallel/serialization/DefaultSerializersLiveTest.js

```diff
@@ -664,3 +658,50 @@ describe('DefaultSerializersLiveTest', function () {
         }
     });
 });
+
+describe('DefaultSerializersLiveTest Arrays', function () {
+    let cluster, client;
+
+    const testFactory = new TestUtil.TestFactory();
+
+    before(async function () {
+        cluster = await testFactory.createClusterForParallelTests();
+        const member = await RC.startMember(cluster.id);
+        client = await testFactory.newHazelcastClientForParallelTests({
+            clusterName: cluster.id,
+            serialization: {
+                defaultNumberType: 'byte'
+            }
+        }, member);
+    });
+
+    beforeEach(async function() {
+        map = await client.getMap(TestUtil.randomString(10));
+    });
+
+    after(async function () {
+        await testFactory.shutdownAll();
+    });
+    it('should get same string values', async function () {
```

Comment:
```suggestion
    it('should serializer string array correctly', async function () {
```

I think this name is better for the test.

Reply:
Fixed